examples/preview/theme/page.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { usePost, WPHead } from '@wpengine/headless';
+
+export default function Page() {
+  const post = usePost();
+
+  return (
+    <>
+      <WPHead />
+
+      <div>
+        {post && (
+          <div>
+            <div>
+              <h5>{post.title}</h5>
```

Comment:
Should we use h1 for the page title?

```suggestion
              <h1>{post.title}</h1>
```

(I know it's h5 in the post template too, but we can correct it separately there.)

Reply:
Nice callout. I'll patch that up.